index.html

```diff
@@ -4805,6 +4805,126 @@ <h2>
 layers, and pad messages to standard lengths.
       </p>
     </section>
+
+    <section>
+      <h2>
+Service Privacy
+      </h2>
+
+      <p>
+This specification provides a mechanism for expressing services that an
+entity might use to interact with a <a>DID subject</a>
+(see <a href="#service-endpoints"></a>). Implementers are advised that
+while service endpoints in a <a>DID Document</a> increase control and agency
+for a <a>DID subject</a>, that they might also become a privacy risk if misused.
+      </p>
+
+      <p>
+For example, consider the range of privacy implications for the following
+service endpoints (ordered by an entity's expectation of no privacy to
+strong privacy protection):
+      </p>
+
+      <ul>
+        <li>
+A governmental DID Document expressing the government's public website.
+        </li>
+        <li>
+A celebrity DID Document expressing all of their official social media profiles.
+        </li>
+        <li>
+An individual's DID Document expressing their professional profile.
+        </li>
+        <li>
+A corporate DID Document expressing contact information for their paying
+customers.
+        </li>
+        <li>
+An individual's DID Document expressing a service endpoint containing their
+email address (questionable idea).
+        </li>
+        <li>
+An individual's DID Document expressing a service endpoint containing their
+personal phone number and home address (bad idea).
+        </li>
+        <li>
+An individual's DID Document expressing endpoints for managing electronic
+health records (bad idea).
```

Comment:
I don't think we need these parentheticals. This is a considerations section -- we should let the reader consider whether or not these things are a good or bad idea. We should just elevate attention to them so they will receive that consideration.